table/tables/tables.go

```diff
@@ -84,6 +84,10 @@ func TableFromMeta(alloc autoid.Allocator, tblInfo *model.TableInfo) (table.Tabl
 			if err != nil {
```

Comment:
Instead of storing `colInfo.GeneratedExprString`, could you store the parsed expr?
So `parser.ParseExpression` is not need any more, it's really some awful code I've ever see.